ompi/mca/hook/demo/hook_demo_fns.c

```diff
@@ -0,0 +1,76 @@
+/*
+ * Copyright (c) 2016      IBM Corporation.  All rights reserved.
+ * $COPYRIGHT$
+ *
+ * Additional copyrights may follow
+ *
+ * $HEADER$
+ */
+
+#include "ompi_config.h"
+
+#include "hook_demo.h"
+
+#define DEBUG_OUTPUT( fn_name )                     \
+    do {                                            \
+        opal_output(0, "hook:demo: " # fn_name );   \
+    } while(0)
+
+void ompi_hook_demo_mpi_initialized_top(int *flag)
+{
+    DEBUG_OUTPUT( mpi_initialized_top );
```

Comment:
No reason. I suppose I just wanted to make work for myself. :) I'll clean that up on the next iteration.